CardanoSharp.Wallet/AddressService.cs

```diff
@@ -147,7 +150,7 @@ addressType switch
                 _ => throw new Exception("Unknown address type")
             };
 
-        private string getPrefixTail(NetworkType networkType) =>
+        public static string GetPrefixTail(NetworkType networkType) =>
             networkType switch
             {
                 NetworkType.Testnet => "_test",
```

Comment:
Depending on which PR is merged first. We may need to update AddressService here or in CIP1854. That PR has two new Address Types which would be using the old name of the function

Reply:
ok this is about to be merged it looks like. once merged, ill update the CIP 1854 PR to reflect changes